control_server.go

```diff
@@ -180,18 +185,22 @@ func (this *ControlServer) HandleUnpause(w http.ResponseWriter, r *http.Request)
 }
 
 func (this *ControlServer) HandleCutover(w http.ResponseWriter, r *http.Request) {
-	vars := mux.Vars(r)
+	cutover, err := cutoverType(r)
+
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+	}
 
-	if vars["type"] == "automatic" {
+	if cutover == "automatic" {
 		this.F.AutomaticCutover = true
-	} else if vars["type"] == "manual" {
+	} else if cutover == "manual" {
```

Comment:
lol at the `"manual"`.. I definitely wasn't careful when I wrote this code originally with regards to how to name different things. We should definitely refactor the cutover process to make things more clear (cc @floriecai )